tests/benchdnn/graph/parser.cpp

```diff
@@ -117,6 +117,35 @@ bool parse_op_attrs(std::vector<std::map<size_t, std::string>> &op_attrs_vec,
     return parse_key_value(op_attrs_vec, op_attrs_str), true;
 }
 
+bool parse_dt(std::vector<dnnl_data_type_t> &dts,
+        std::vector<std::map<size_t, dnnl_data_type_t>> &dt_maps,
+        const char *str, const std::string &option_name) {
+    std::string dts_str;
+    if (!parse_string(dts_str, str, option_name)) return false;
+
+    if (dts_str.find(":") == std::string::npos) {
+        // format like --dt=f32,bf16,f16
+        const std::vector<dnnl_data_type_t> def_dt = {dnnl_data_type_undef};
+        parser::parse_dt(dts, def_dt, str, option_name);
+    } else {
```

Comment:
Based on this logic it looks like the following will work:
```
--dt=f16 --dt=0:f32+... --case=some_case
```
`if` branch will update `dt`, then `else` branch will update `dt_map`, then `dt_rewrite` will update the whole graph, and then `dt_map_rewrite` will update specific IDs.
Is it an expected flow?
If yes, should it be properly documented?
If no, then it probably should be restricted here by additional checking whether the other object was updated or not.

Reply:
Good catch! I did not expect the two --dt flavors would be specified in a single command line. I just added a check in the `bench()` function to error out.